spec/factories/comments.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :comment do
+    author { Faker::Name.name }
+    body { Faker::ChuckNorris.fact }
+    recipe
```

Comment:
```suggestion
    recipe
    author { Faker::Name.name }
    body { Faker::ChuckNorris.fact }
```
Associations should be before attributes